newIDE/app/src/AssetStore/AssetStoreContext.js

```diff
@@ -236,9 +240,10 @@ export const AssetStoreStateProvider = ({
         isLoading.current = false;
       })();
     },
-    [assetShortHeadersById, isLoading]
```

Comment:
Can you remind me why assetShortHeadersById is not useful?

Reply:
There used to be a check to ensure the assets are loaded only once for the whole session.
But now that we change the environment, we want the callback to run again so we don't care if there were already results or not.